integration-tests/15_achievements.ts

```diff
@@ -0,0 +1,1043 @@
+            userId: user.userID,
+        },
+    });
+    assert.ok(student_onboarding_2);
+
+    // Create Certificate of Conduct
+    const newDate = JSON.stringify(new Date());
+    await adminClient.request(`
+        mutation CreateCertificateOfConduct {
+            certificateOfConductCreate(
+                dateOfInspection: ${newDate},
+                dateOfIssue: ${newDate},
+                criminalRecords: false,
+                studentId: ${student.student.id},
+            )
+        }
+    `);
+    const student_onboarding_3 = await prisma.user_achievement.findFirst({
+        where: {
+            group: 'student_onboarding',
+            groupOrder: 4,
+            achievedAt: { not: null },
+            userId: user.userID,
+        },
+    });
+    const student_onboarding_4 = await prisma.user_achievement.findFirst({
+        where: {
+            group: 'student_onboarding',
+            groupOrder: 5,
+            userId: user.userID,
+        },
+    });
+    assert.ok(student_onboarding_3);
+    assert.ok(student_onboarding_4);
+});
+
+void test('Reward pupil onboarding achievement sequence', async () => {
+    await adminClient.request(`mutation ResetRateLimits { _resetRateLimits }`);
+    await createPupilOnboardingTemplates();
+
+    // Verify Email
+    const { pupil } = await createNewPupil();
+    const user = await getUser(pupil.userID);
+
+    const pupil_onboarding_1 = await prisma.user_achievement.findFirst({
+        where: {
+            group: 'pupil_onboarding',
+            groupOrder: 1,
+            achievedAt: { not: null },
+            userId: user.userID,
+        },
+    });
+    assert.ok(pupil_onboarding_1);
+    // Screening
+    await adminClient.request(`
+        mutation RequestScreening { pupilCreateScreening(pupilId: ${pupil.pupil.id})}
+    `);
+    const pupil_onboarding_2 = await prisma.user_achievement.findFirst({
+        where: {
+            group: 'pupil_onboarding',
+            groupOrder: 3,
+            achievedAt: { not: null },
+            userId: user.userID,
+        },
+    });
+    const pupil_onboarding_3 = await prisma.user_achievement.findFirst({
+        where: {
+            group: 'pupil_onboarding',
+            groupOrder: 4,
+            userId: user.userID,
+        },
+    });
+    assert.ok(pupil_onboarding_2);
+    assert.ok(pupil_onboarding_3);
+});
+
+void test('Reward student conducted match appointment', async () => {
+    await adminClient.request(`mutation ResetRateLimits { _resetRateLimits }`);
+    await createStudentConductedMatchAppointmentTemplates();
+    const { student, client } = await studentOne;
+    const user = await getUser(student.userID);
+
+    const uuid = generateUUID();
+    const match = await prisma.match.create({
+        data: {
+            uuid,
+            source: 'matchedinternal',
+            matchPool: 'lern-fair-now',
+            studentId: student.student.id,
+            pupilId: 1,
+        },
+    });
```

Comment:
You could try to use `CreateManualMatch` here

https://github.com/corona-school/backend/blob/7bb6eed4f8fe3dd1107ac2315898c56a9eaa4117/integration-tests/03_matching.ts#L83-L87